kubeflow/trainer/backends/kubernetes/backend_test.py

```
@@ -1395,3 +1399,306 @@ def test_get_job_events(kubernetes_backend, test_case):
+        patch("kubernetes.client.CoreV1Api"),
+    ):
+        mock_get_client.return_value = mock_api_client
+
+        cfg = KubernetesBackendConfig(
+            auth_method="oidc",
+            oidc_issuer="https://issuer.example.com",
+            oidc_client_id="client-id",
+            oidc_client_secret="client-secret",
+            use_device_flow=True,
+        )
+        backend = KubernetesBackend(cfg)
```

Ruff (0.14.10)

Local variable `backend` is assigned to but never used

Remove assignment to unused variable `backend`

(F841)